Tests/MaterialLib/TestMPLPermeabilityMohrCoulombFailureIndexModel.cpp

```diff
@@ -0,0 +1,104 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * Created on June 8, 2020, 9:17 AM
+ */
+
+#include <gtest/gtest.h>
+
+#include <Eigen/Eigen>
+#include <boost/math/constants/constants.hpp>
+
+#include "MaterialLib/MPL/Medium.h"
+#include "MaterialLib/MPL/Properties/CreatePermeabilityMohrCoulombFailureIndexModel.h"
+#include "MaterialLib/MPL/Properties/PermeabilityMohrCoulombFailureIndexModel.h"
+#include "MaterialLib/MPL/Utils/FormEigenTensor.h"
+#include "MathLib/KelvinVector.h"
+#include "ParameterLib/ConstantParameter.h"
+#include "TestMPL.h"
+#include "Tests/TestTools.h"
+
+TEST(MaterialPropertyLib, PermeabilityMohrCoulombFailureIndexModel)
+{
+    ParameterLib::ConstantParameter<double> k0 =
+        ParameterLib::ConstantParameter<double>("k0", 1.e-20);
```

Comment:
```suggestion
    ParameterLib::ConstantParameter<double> k0("k0", 1.e-20);
```

Reply:
and const

Reply:
Updated.

Reply:
Changed. https://github.com/ufz/ogs/pull/2989/commits/df4733f08f7e3f8fbbfe9f8c114868e27e35bd4a#diff-bb5a3a2f4bc492d8735683d3c0ef44a7R28